rrfs-test/validated_yamls/templates/obtype_config/msonet_airTemperature_188.yaml

```diff
@@ -52,29 +71,15 @@
              is_in: 188
            action:
              name: assign error
-             error parameter: 4.0  # K
-           defer to post: true
-
-         ## Ajusted error after initial assignment (qcmod.f90)
-         #- filter: Perform Action
-         #  filter variables:
-         #  - name: airTemperature
-         #  where:
-         #  - variable: ObsType/airTemperature
-         #    is_in: 188
-         #  action:
-         #    name: inflate error
-         #    inflation variable:
-         #      name: ObsFunction/ObsErrorFactorConventional
-         #      options:
-         #        test QCflag: PreQC
-         #        test QCthreshold: 3
-         #        inflate variables: [airTemperature]
-         #        pressure: MetaData/pressure
-         #        distance threshold: -1.
-         #  defer to post: true
+             error function:
+               name: ObsFunction/ObsErrorModelStepwiseLinear
+               options:
+                 xvar:
+                   name: MetaData/pressure
+                 xvals: [110000, 105000, 100000, 95000, 90000, 85000, 80000, 75000, 70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000, 500, 400, 300, 200, 100, 0]
+                 errors: [4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]
```

Comment:
I might have missed this in previous discussion, but how is this different from the previous version that sets a constant error of 4 K to all obs?

Reply:
It isn't. I was just doing that for consistency between all yamls. So side-by-side comparisons or when I create a new yaml based on the msonet obs there will be less to change. It's effectively the same, but just a matter of my own preference for managing the yamls.

Reply:
Okay, makes sense. I guess this is similar to how things are defined in the GSI `errtable` too with a profile set for each obtype including mesonet